npm/serpen/bin/serpen.js

```
@@ -0,0 +1,158 @@
+    try {
+        // Check for musl-specific files/processes
+        if (existsSync('/lib/ld-musl-x86_64.so.1') ||
+            existsSync('/lib/ld-musl-aarch64.so.1') ||
+            existsSync('/usr/lib/libc.musl-x86_64.so.1') ||
+            existsSync('/usr/lib/libc.musl-aarch64.so.1')) {
+            return true;
+        }
+
+        // Try to read /proc/version for musl indicators
+        if (existsSync('/proc/version')) {
+            const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)